### DISCUSSION TITLE
Could Inconsolata be added as the default monospaced font?

### DISCUSSION THREAD

**Discussion by jmg-duarte** [joaomlourenco/novathesis#191] [Q&A] [answered]
The title says it all. As a support, consider the following.

## Before
![image](https://user-images.githubusercontent.com/15343819/121200778-e856f180-c86b-11eb-9300-3a04931bd688.png)

## After
![image](https://user-images.githubusercontent.com/15343819/121200977-10465500-c86c-11eb-8227-4ab395f5b86d.png)

As far as I know, making it default can be done by adding:

```latex
\usepackage{inconsolata}
```

**Comment by joaomlourenco** [comment, marked as answer]
Converted to issue.
